### PR TITLE
makefile fixes

### PR DESCRIPTION
**Pull request by tchughesiv**
Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -73,7 +73,7 @@ fmt:
 
 # Run go vet against code
 vet:
```
Comment (font) [inline Makefile:75]: Should we add it to the `go fmt` make recipe?

**Review by font — COMMENTED** [review]
Thanks @tchughesiv !

File: Makefile (hunk 1 above)
Comment (tchughesiv) [reply Makefile:75]: in go 1.13 fmt throws an error - https://github.com/golang/go/issues/27841

if we don't care about supporting 1.13, i think we can add it yes.

Comment (font) [reply Makefile:75]: Ah, ok. We can hold off adding it as I've updated the go version dependency in `go.mod` in some of the installation work.

**Review by font — APPROVED** [review]

**Pull request merged**